ably-ios/ARTAuthTokenParams.m

```diff
@@ -19,7 +20,7 @@ @implementation ARTAuthTokenParams
 
 - (instancetype)init {
     if (self = [super init]) {
-        _ttl = 60 * 60;
+        _ttl = [ARTDefault ttl] * 60; //seconds
```

Comment:
The TTL is in miliseconds in the API, yet it appears the value here is in seconds meaning the intended `TTL` will be incorrect.  I would like to see a test that requests a token valid for 24 hours, then checks that the token does indeed expire the following day 24 hours later (or within a minute of that).